packages/cadl-ranch-specs/http/projection/main.cadl

```diff
@@ -0,0 +1,59 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+
+@doc("Projection")
+@supportedBy("dpg")
+@scenarioService("/projection")
+namespace Projection;
+
+model Project {
+  @projectedName("json", "codegen")
+  producedBy?: string;
+
+  @projectedName("client", "createdBy")
+  builtfrom?: string;
+
+  @projectedName("client", "DontUseMeAsName") // Should be ignored
+  @projectedName("csharp", "MadeFor")
```

Comment:
Should we have a different name for each languages just to make sure it is respected. Like `madeForCS`. At that point would also respect the naming convention of each language